pennylane/devices/__init__.py

```diff
@@ -168,8 +169,9 @@ def execute(self, circuits, execution_config = qml.devices.DefaultExecutionConfi
 from .null_qubit import NullQubit
 from .default_qutrit import DefaultQutrit
 from .default_qutrit_mixed import DefaultQutritMixed
-from .._device import Device as LegacyDevice
-from .._device import DeviceError
+from ._device import Device as LegacyDevice
+from ._qubit_device import QubitDevice
+from ._qutrit_device import QutritDevice
```

Comment:
As discussed in person, I am more a partisan of absolute imports, and since we are changing these lines already maybe it would be a good chance to make this transition. Not a blocker though. Interested to hear both of your opinions @mudit2812 @Shiro-Raven

Reply:
You don't use absolute imports in __init__ file